test/integration/authorise_application_test.rb

```diff
@@ -74,14 +74,18 @@ class AuthoriseApplicationTest < ActionDispatch::IntegrationTest
     end
 
     assert_redirected_to_application @app
-    assert_kind_of Doorkeeper::AccessGrant, Doorkeeper::AccessGrant.find_by(resource_owner_id: @user.id)
+    assert_access_granted @user
   end
 
   def assert_redirected_to_application(app)
     assert_match(/^#{app.redirect_uri}/, current_url)
     assert_match(/\?code=/, current_url)
   end
 
+  def assert_access_granted(user)
+    assert_kind_of Doorkeeper::AccessGrant, Doorkeeper::AccessGrant.find_by(resource_owner_id: user.id)
```

Comment:
Minor: It might be nice to add a custom message to this assertion.

Reply:
I've addressed this in c440cd73a8b9f3b1136ab312d7d1e06addb4fcf9.